### DISCUSSION TITLE
Spectrum AR637T

### DISCUSSION THREAD

**Discussion by Boebe62** [FrSkyRC/ETHOS-Feedback-Community#3936] [Q&A]
I bought a E-flite plane with a build in AR637T receiver. I can bind the plane with the 4-1 multiprotocol module. Now I need to assign a switch to change the gyro mode from SAFE to AS3X. I followed the E-flite manual but no luck.
Anyone out here with the golden tip?

**Comment by Crunch69** [comment]
go to rcgroups
